react/features/local-recording/components/LocalRecordingInfoDialog.js

```diff
@@ -0,0 +1,332 @@
+                    </li>)
+                    /* eslint-enable no-extra-parens */
+                )}
+            </ul>
+        );
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { isModerator, encodingFormat, isOn, t } = this.props;
+        const { durationString } = this.state;
+
+        return (
+            <div
+                className = 'info-dialog' >
+                <div className = 'info-dialog-column'>
+                    <h4 className = 'info-dialog-icon'>
+                        <i className = 'icon-info' />
+                    </h4>
+                </div>
+                <div className = 'info-dialog-column'>
+                    <div className = 'info-dialog-title'>
+                        { t('localRecording.localRecording') }
+                    </div>
+                    <div>
+                        <span className = 'info-label'>
+                            {`${t('localRecording.moderator')}:`}
+                        </span>
+                        <span className = 'spacer'>&nbsp;</span>
+                        <span className = 'info-value'>
+                            { isModerator
+                                ? t('localRecording.yes')
+                                : t('localRecording.no') }
+                        </span>
+                    </div>
+                    { isOn && <div>
+                        <span className = 'info-label'>
+                            {`${t('localRecording.duration')}:`}
+                        </span>
+                        <span className = 'spacer'>&nbsp;</span>
+                        <span className = 'info-value'>
+                            { durationString }
+                        </span>
+                    </div>
+                    }
+                    {isOn
+                    && <div>
+                        <span className = 'info-label'>
+                            {`${t('localRecording.encoding')}:`}
+                        </span>
+                        <span className = 'spacer'>&nbsp;</span>
+                        <span className = 'info-value'>
+                            { encodingFormat }
+                        </span>
+                    </div>
+                    }
+                    {
+                        isModerator
+                        && <div>
+                            <div>
+                                <span className = 'info-label'>
+                                    {`${t('localRecording.participantStats')}:`}
+                                </span>
+                            </div>
+                            { this.renderStats() }
+                        </div>
+                    }
+                    {
+                        isModerator
+                            && <div className = 'info-dialog-action-links'>
```

Comment:
Currently I am just using the CSS in `info-dialog.scss`. Which is probably not intended for general use outside of `InfoDialog`.

Reply:
Probably not but you can break them out into a separate css file. For example, they're inline dialogs so you can create an inline-dialog.scss and refactor for reuse.